addons/point_of_sale/models/stock_warehouse.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+
+class Warehouse(models.Model):
+    _inherit = "stock.warehouse"
+    
+    pos_type_id = fields.Many2one('stock.picking.type', string="PoS type")
```

Comment:
shitty label is shitty (I know, you took example on stock - I prefer the example in mrp though)
let's use "Point of Sale Operation Type" instead